pkg/occlient/occlient.go

```diff
@@ -1389,22 +1389,46 @@ func (c *Client) CreateServiceInstance(componentName string, componentType strin
 	return nil
 }
 
+type ServiceList struct {
+	Name     string
+	PlanList []string
+}
+
 // GetClusterServiceClassExternalNames returns the names of all the cluster service
 // classes in the cluster
-func (c *Client) GetClusterServiceClassExternalNames() ([]string, error) {
-	var classNames []string
+func (c *Client) GetClusterServiceClassExternalNames() ([]ServiceList, error) {
```

Comment:
There is no unit test for this function :-(

Reply:
added :tada:

Reply:
Maybe rename the function to something like `GetClusterServiceClassExternalNamesAndPlans()` and also please modify the comments and add that it returns their plan names too